src/hyperlight_guest_capi/src/lib.rs

```diff
@@ -3,6 +3,10 @@
 
 extern crate alloc;
 
+// TODO(danbugs): this is needed so the panic handler is actually brought in.
```

Comment:
Is this actioned in a later commit?

Reply:
I addressed all TODOs by the end of the PR.